src/sentry/api/endpoints/organization_events_meta.py

```diff
@@ -126,6 +126,14 @@ def get(self, request: Request, organization) -> Response:
         second_bound = request.GET.get("secondBound")
         upper_bound = request.GET.get("upperBound")
         column = request.GET.get("column", "span.self_time")
+        selected_columns = request.GET.getlist("field", [],) + [
+            "project",
+            "transaction.id",
+            column,
+            "timestamp",
+            "span_id",
+            "profile_id",
+        ]
```

Comment:
hmm, actually if we're concatenating the list, can we call it `additionalFields` or something more accurate?
```suggestion
        selected_columns = [
            "project",
            "transaction.id",
            column,
            "timestamp",
            "span_id",
            "profile_id",
        ]
        
        additional_columns = request.GET.getlist("additionalFields", [])
```
and below
...
selected_columns=selected_columns + additional_columns,

Reply:
Yeah I think that's a good point 👍, I'll change it to additionalFields